File: C++/count-the-number-of-consistent-strings.cpp
class Solution {
public:
    int countConsistentStrings(string allowed, vector<string>& words) {
        set<char> test(allowed.begin(), allowed.end());
        
        int count = 0;
        for (string word: words) {
            bool ok = true;
            for (char c: word) {
                if (test.count(c) == 0) {
                    ok = false;
                    break;
                }
            }
            if (ok) count++;
        }
        return count;
    }
};

File: Go/count-the-number-of-consistent-strings.go
func countConsistentStrings(allowed string, words []string) int {
	a := [26]bool{}
	for _, c := range allowed {
		a[c-'a'] = true
	}
	count := 0
outer:
	for _, w := range words {
		for _, c := range w {
			if !a[c-'a'] {
				continue outer
			}
		}
		count++
	}
	return count
}